.github/workflows/test-application.yaml

```diff
@@ -75,6 +75,10 @@ jobs:
                   extensions: "pdo, pdo_sqlite, pdo_mysql, mysql, pdo_pgsql"
                   tools: 'composer:v2'
 
+            - name: Require dependencies
+              if: ${{ matrix.php-version == '8.0' }}
+              run: composer require "psr/simple-cache:2.*" --no-update --dev
```

Comment:
can you please add a comment why we need this? (to prevent the conflict symfony 5 if i remember correctly)

Reply:
thanks